Maestro/subscriptions.json

```diff
@@ -1114,34 +1022,6 @@
         }
       }
     },
-    // Update 2.1 dependencies in the dotnet-docker repo's nightly branch
-    {
-      "triggerPaths": [
-        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/product/cli/release/2.1/packages+installers.semaphore"
```

Comment:
Is this just a part of removing 2.1 daily build stuff because we don't really do those anymore?

/cc @MichaelSimons @mthalman